CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -2316,13 +2349,28 @@ public let DECL_NODES: [Node] = [
       Child(
         name: "BindingSpecifier",
         deprecatedName: "BindingKeyword",
-        kind: .token(choices: [.keyword(text: "let"), .keyword(text: "var"), .keyword(text: "inout")])
+        kind: .token(choices: [.keyword(text: "let"), .keyword(text: "var"), .keyword(text: "inout")]),
+        documentation: """
+          The specifier that defines the type of the variables declared (`let` or `var`).
+          """
       ),
       Child(
         name: "Bindings",
-        kind: .collection(kind: .patternBindingList, collectionElementName: "Binding")
+        kind: .collection(kind: .patternBindingList, collectionElementName: "Binding"),
+        documentation: """
+          The pattern bindings that define the actual variables.
+
+          The pattern bindings contain the declared variables’ names, their types,
+          initializers and accessors.
+
+          A variable declaration can contain multiple pattern bindings, because it’s possible
+          to define multiple variables after a single `let` keyword, for example
+
+          ```swift
+          let x: Int = 1, y: Int = 2
+          ```
```

Comment:
I've seen that some SwiftUI [documentation](https://developer.apple.com/documentation/swiftui/view/labelstyle(_:)) also have code examples on how to use. 

What do you think of also adding it here to make it more explicit for consumers on how to make the example you provided?

Reply:
I'm not entirely sure what you mean.

Reply:
I mean added the example that constructs `let x: Int = 1, y: Int = 2`.
I think it would make it easier to consume for the user to see actual examples in the documentation. 

Maybe something like

```
A variable declaration can contain multiple pattern bindings, because it’s possible
          to define multiple variables after a single `let` keyword, for example
          
```swift
VariableDeclSyntax(bindingSpecifier: .keyword(.let)) {
            PatternBindingSyntax(pattern: IdentifierPatternSyntax(identifier: .identifier("x")),
                                 typeAnnotation: TypeAnnotationSyntax(type: IdentifierTypeSyntax(name: .identifier("Int"))),
                                 initializer: InitializerClauseSyntax(value: IntegerLiteralExprSyntax(1))
            )

            PatternBindingSyntax(pattern: IdentifierPatternSyntax(identifier: .identifier("y")),
                                 typeAnnotation: TypeAnnotationSyntax(type: IdentifierTypeSyntax(name: .identifier("Int"))),
                                 initializer: InitializerClauseSyntax(value: IntegerLiteralExprSyntax(2))
            )
        }
```
          
generates          

          ```swift
          let x: Int = 1, y: Int = 2
          ```
```

Reply:
Oh, I don’t think that would be helpful in this context. The purpose of this comment is to educate the reader why a single variable declaration might have multiple pattern bindings and thus explain why the API seems so complicated. I very much doubt that anyone wants to generate such a declaration.